calour/training.py

```diff
@@ -0,0 +1,100 @@
+'''
+filtering (:mod:`calour.filtering`)
+===================================
+
+.. currentmodule:: calour.filtering
+
+Functions
+^^^^^^^^^
+.. autosummary::
+   :toctree: generated
+
+   filter_by_data
+   filter_training
+'''
```

Comment:
need to fix to correct doc :)

Reply:
I know. "WIP" means work in progress. so there is lots of things subject to changes. will fix this later after I decided the names, implementation, etc.